device_coupler/testing/test_device_coupler.out

```diff
@@ -1,7 +1,9 @@
+9a:02:57:1e:8f:01
+=================
 RESULT pass base.startup.dhcp
 RESULT skip base.switch.ping
 RESULT pass connection.base.target_ping
-RESULT pass security.nmap.ports
+RESULT fail security.nmap.ports
```

Comment:
so this doesn't seem right?  You mentioned something about this when we chatted... would be a good idea to resolve the underlying issue first if there is one.

Reply:
This delta shows up because the earlier device_coupler blocked/did not route certain kinds of traffic. This also resulted in tests like ncrack and dot1x. not working. This change makes sure all tests work and takes care of that. The fail result is actually what is expected and is the same as what is seen in the output of test_aux.